src/local-nav/local-nav-dropdown/local-nav-dropdown.tsx

```diff
@@ -127,6 +131,16 @@ const Component = (
         };
     }, []);
 
+    useEffect(() => {
+        listItemRefs.current = Array(items.length).fill(null);
```

Comment:
this might end up unintentionally clearing the existing refs when the options are updated asynchronously, I think this logic can be skipped for now